### ISSUE TITLE
FTB Ultimine outlines not showing

### ISSUE THREAD

**Issue by kaweepatinn1** [FTBTeam/FTB-Ultimine#39]
the outlines used to show, but now they don't. I have hunger, and the veinmining works but the white outlines do not show. Any fix?

**Comment by MaxNeedsSnacks** [comment]
You probably have `renderOutline` disabled, or are using a 1.12 version which doesn't have the outline rendering because it was way too laggy to implement

**Comment by AefRal** [comment]
So the outlines stopped showing randomly.
I mean within the same world, without exiting, was playing and had them. Later I went to go mining again and they were gone. Full on food and health.

Playing Team Rustic 4, added FTB Ultimine 1802.3.3-build.55, and FTB Library 1802.3.6-build.115.
Also Optifine and shaders, though don't think that's the problem.

**Comment by MichaelHillcox** [comment]
It's optifine, use an alternative:

Here are some alternatives for optifine instead
Performance - https://github.com/NordicGamerFE/usefulmods/tree/main/Performance
Dynamic Lighting - https://www.curseforge.com/minecraft/mc-mods/lucent
Zoom - https://www.curseforge.com/minecraft/mc-mods/ezzoom
Shaders - https://github.com/bradleyq/mc_vanilla_shaders/releases/tag/v2.3 or https://www.curseforge.com/minecraft/mc-mods/oculus

https://gist.github.com/CarbonGhost/748742abff3a7386940f1c5de42afe69#why-ditch-optifine for more info on why I've said to not use optifine.

tl;dr, it breaks mods because it's written without mods in mind.

**Comment by AefRal** [comment]
Ah, thank you.
